pkg/master/import_known_versions_test.go

```diff
@@ -85,6 +85,7 @@ var typesAllowedTags = map[reflect.Type]bool{
 	reflect.TypeOf(metav1.GroupVersionKind{}):     true,
 	reflect.TypeOf(metav1.GroupVersionResource{}): true,
 	reflect.TypeOf(metav1.Status{}):               true,
+	reflect.TypeOf(runtime.Unknown{}):             true,
```

Comment:
This is surprising.

Reply:
This is because [Event](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/apis/audit/types.go#L121) needs to be a known type in the scheme but the nested request and response objects are of type `runtime.Unknown` which has tags. Only other way I could think around this piece would be to have the object be in its own api.

Reply:
We have admission and admissionregistration probably for the same reason. I think splitting served types from transient review/response types would be a good idea.

Reply:
ok @sttts went ahead and separated out the api, let me know what you think